docs/source/quickstart.mdx

```diff
@@ -1,296 +1,550 @@
+2. **Test the connection** by asking: "What MCP tools do you have available?"
+3. **Verify GraphQL operations** are listed as available tools
+4. **Test a query** using one of your configured operations
 
 ## Troubleshooting
 
-### Common Issues
-
-#### MCP Server Won't Start
-- **Error**: "Port 5000 is already in use"
-  - Solution: Kill any existing processes using port 5000 or specify a different port with the `transport.port` option or `APOLLO_MCP_TRANSPORT__PORT` env variable
-- **Error**: "Failed to load supergraph configuration"
-  - Solution: Verify you're running the command from the repo root directory
-  - Solution: Check that the path to `supergraph.yaml` is correct
-
-#### MCP Inspector Connection Issues
-- **Error**: "Failed to connect to server"
-  - Solution: Ensure the MCP server is running (check terminal output)
-  - Solution: Verify you're using the correct URL (`http://127.0.0.1:5000/mcp`)
-  - Solution: Check if your firewall is blocking the connection
-
-#### Claude Desktop Issues
-- **Problem**: Claude doesn't recognize the tools
-  - Solution: Verify the config file path is correct for your OS
-  - Solution: Ensure the JSON is properly formatted (no trailing commas)
-  - Solution: Try restarting Claude Desktop completely
-- **Problem**: "Connection refused" errors
-  - Solution: Check if the MCP server is still running
-  - Solution: Verify the port numbers match in both the server and Claude config
-- **Problem**: "MCP thespacedevs: Server disconnected" errors
-  - Solution: Uninstall older versions of Node. `mcp-remote` only works with Node v18 or later.
-  - Solution: Restart Claude Desktop
-
-#### GraphQL Operation Issues
-- **Error**: "Operation not found"
-  - Solution: Verify the operation files exist in the specified path
-  - Solution: Check that the operation names match exactly
-- **Error**: "Schema validation failed"
-  - Solution: Ensure your GraphQL operations match the schema
-  - Solution: Check for syntax errors in your operation files
+<details>
+<summary>Common Issues</summary>
+
+**Client doesn't see tools:**
+- Ensure you restarted your MCP client after configuration
+- Verify the Apollo MCP Server is running (`rover dev` command)
+- Check port numbers match between server and client config
+
+**Connection refused errors:**
+- Confirm the server is running on the correct port (default: 5050)
+- Verify firewall settings allow connections to localhost:5050
+- For remote connections, ensure the host is set to `0.0.0.0` in your config
+
+**GraphQL operation errors:**
+- Verify your operations files exist in the specified paths
+- Check that operation names match your GraphQL schema
+- Ensure your GraphQL endpoint is accessible
+
+**Authentication issues:**
+- Verify environment variables are properly set
+- Check that your GraphQL endpoint accepts the provided headers
+- Test your GraphQL endpoint directly with tools like GraphiQL
+</details>
+<details>
+<summary>Getting Help</summary>
```

Comment:
We have the exact same content later on in the additional resource section.

Trouble shooting should be actionable - I think it's more appropriate in the later spot.